docs/users-guide/01-what-is-metabase.md

```diff
@@ -49,7 +49,3 @@ To make a dashboard or pulse, click the plus (+) icon in the top-right of the ma
 ![Search results](images/basic-exploration/search-results.gif)
 
 The search bar at the top of the screen helps you find tables, dashboards, collections, saved questions, metrics, segments, and pulses in an instant.
-
-## A primer on databases
```

Comment:
Should we instead link to https://www.metabase.com/docs/latest/users-guide/03-basic-exploration.html?